src/components/DocumentSelector/assets/icon-passport-hover.svg

```diff
@@ -1,34 +1,34 @@
-                <path d="M23.873,44.8534826 L17.946,44.8534826 L17.946,44.8534826 C18.066,47.0944826 18.713,49.1964826 19.766,51.0384826 L24.954,51.0334826 C24.321,49.1084826 23.945,47.0274826 23.873,44.8534826 L23.873,44.8534826 Z" id="Shape" fill="#20c7a0"></path>
-                <path d="M24.953,37.1494826 L19.765,37.1434826 C18.713,38.9854826 18.066,41.0864826 17.945,43.3284826 L23.872,43.3284826 C23.945,41.1534826 24.321,39.0744826 24.953,37.1494826 L24.953,37.1494826 Z" id="Shape" fill="#20c7a0"></path>
-                <path d="M26.56,37.1444826 C25.872,39.0744826 25.465,41.1514826 25.389,43.3274826 L31.183,43.3274826 L31.183,37.1444826 L26.56,37.1444826 L26.56,37.1444826 Z" id="Shape" fill="#20c7a0"></path>
-                <path d="M28.695,57.7224826 C27.428,56.2534826 26.327,54.4804826 25.527,52.5684826 L20.77,52.5684826 C22.623,55.0114826 25.424,56.9844826 28.695,57.7224826 L28.695,57.7224826 Z" id="Shape" fill="#20c7a0"></path>
-                <path d="M28.695,30.4604826 C25.422,31.1984826 22.623,33.1714826 20.769,35.6144826 L25.526,35.6144826 C26.327,33.7024826 27.428,31.9284826 28.695,30.4604826 L28.695,30.4604826 Z" id="Shape" fill="#20c7a0"></path>
-                <path d="M31.183,35.6154826 L31.183,30.0964826 C29.551,31.5564826 28.173,33.4414826 27.176,35.6154826 L31.183,35.6154826 L31.183,35.6154826 Z" id="Shape" fill="#20c7a0"></path>
-                <path d="M59.117,0.0914825785 L4.765,0.0914825785 C2.62,0.0914825785 0.882,1.82948258 0.882,3.97348258 L0.882,84.2084826 C0.882,86.3534826 2.62,88.0914826 4.765,88.0914826 L59.118,88.0914826 C61.262,88.0914826 63,86.3534826 63,84.2094826 L63,3.97348258 C63,1.82948258 61.262,0.0914825785 59.117,0.0914825785 L59.117,0.0914825785 Z M31.941,28.5624826 C40.517,28.5624826 47.47,35.5154826 47.47,44.0914826 C47.47,52.6674826 40.517,59.6204826 31.941,59.6204826 C23.365,59.6204826 16.412,52.6674826 16.412,44.0914826 C16.412,35.5154826 23.365,28.5624826 31.941,28.5624826 L31.941,28.5624826 Z" id="Shape" fill="#20c7a0"></path>
-                <rect id="Rectangle-78" fill="#F1F8FE" x="11.9049644" y="72.3217781" width="41" height="1" rx="0.5"></rect>
-                <rect id="Rectangle-78" fill="#F1F8FE" x="11.9049644" y="77.4493031" width="41" height="1" rx="0.5"></rect>
-                <rect id="Rectangle-80" fill="#F1F8FE" x="16" y="8.61571707" width="31" height="2.50948416" rx="1.25474208"></rect>
-                <rect id="Rectangle-80" fill="#F1F8FE" x="21.5760711" y="14.9430433" width="20.7543312" height="1" rx="0.5"></rect>
+    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="icons/passport/hover" transform="translate(-29.000000, -20.000000)" fill-rule="nonzero">
+            <g id="passport_hover" transform="translate(29.000000, 20.000000)">
```

Comment:
Not really super important, but it's definitely worth having a look at the SVGs that Sketch automatically generates, as they're normally silly

All of these extra `g`s in this case are doing nothing (except possibly the first `<g fill="none">`

If you read it, you'll see it's doing stuff like translating +29px, followed immediately by -29px.

But as I say, not really that important

Reply:
do you know a good svg optimizer?

Reply:
I've used this one https://jakearchibald.github.io/svgomg/
Seems to be working fine